lightly/__init__.py

```diff
@@ -79,40 +79,25 @@
 
 import os
 
+# see if torchvision vision transformer is available
 try:
-    # See (https://github.com/PyTorchLightning/pytorch-lightning)
-    # This variable is injected in the __builtins__ by the build
-    # process. It used to enable importing subpackages of skimage when
-    # the binaries are not built
-    __LIGHTLY_SETUP__
-except NameError:
-    __LIGHTLY_SETUP__ = False
-
-
-if __LIGHTLY_SETUP__:
-    # setting up lightly
-    msg = f"Partial import of {__name__}=={__version__} during build process."
-    print(msg)
-else:
```

Comment:
This code was necessary to figure out if lightly is imported from `setup.py` or from somewhere else. If it was imported from `setup.py`, then we didn't want to run the version checks because they failed due to missing dependencies. Dependencies were missing because they are not yet available at when `setup.py` runs.

This code has been replaced by the `load_version` function added to `setup.py`. See its docstring for more info.